handler.py

```diff
@@ -0,0 +1,30 @@
+import json
+import datetime
+import logging
+
+from util.s3_helper import send_to_s3
+from lib.target import Target
+from scanners.httpobs_scanner import HTTPObservatoryScanner
+from util.host_picker import Randomizer
+from util.response import Response
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def runObsScan(event, context):
+
+    # Test out S3 upload capability
+    url = 'https://raw.githubusercontent.com/mozilla/vautomator-serverless/scheduled-scans/hostlist.json'
+    randomizer = Randomizer(url)
+    scanner = HTTPObservatoryScanner()
+    destination = Target(randomizer.next())
```

Comment:
can 'destination' be just 'target'?  I feel like it's best when we call things by one name to avoid confusion for the future

Reply:
Yeah, no worries, will change.